work_test.go

```diff
@@ -450,10 +456,9 @@ func TestJobDone(t *testing.T) {
 
 	// make sure conn was returned to pool
 	stat := c.pool.Stat()
-	total, available := stat.CurrentConnections, stat.AvailableConnections
-	if total != available {
-		t.Errorf("want available=total, got available=%d total=%d", available, total)
-	}
+	require.Equal(t, int32(5), stat.MaxConns())
+	require.Equal(t, int32(1), stat.TotalConns())
+	require.Equal(t, int32(1), stat.IdleConns())
```

Comment:
down here you can see that when the connection is returned the pool releases one connection so it only has one idle one.